repository/src/main/resources/db/migration/V20210820__CCD-1323_change_of_increment_size.sql

```diff
@@ -0,0 +1,6 @@
+ALTER SEQUENCE public.event_post_state_id_seq INCREMENT BY 50;
```

Comment:
A minor correction may require in the file name - V20210820_1323__CCD-1323_change_of_increment_size.sql